cli/cmd_root.go

```diff
@@ -29,6 +29,7 @@ func init() {
 	rootCmd.AddCommand(cleanCmd)
 
 	// workspace
+	cmdAdd.Flags().Bool("explicit-protocol", false, "Set to repository protocol explicit to https/ssh depending on the given url")
```

Comment:
As there is just this one flag on the add cmd a simpler version would also do.
"explicit-protocol" => "plain" ?

Also
```suggestion
	cmdAdd.Flags().Bool("plain", false, "Don't infer contrary protocol url (git/https) from scheme")
```

Reply:
changed to `plain`